doc/overview.qbk

```diff
@@ -0,0 +1,46 @@
+[/
+    Copyright (c) 2021 Alan de Freitas (alandefreitas @ gmail.com)
+
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+    Official repository: https://github.com/boostorg/utility
+]
+
+[section Overview]
+[block'''<?dbhtml stop-chunking?>''']
+
+Boost.Utility is a collection of small, useful, and general-purpose components for language support.
+
+Over time,
+
+* several components have been moved to more appropriate Boost libraries and
+* many of these components had variants accepted into the C++ standard
+
+When the component has moved to another Boost library, Boost.Utility headers still lead to the appropriate headers
+in other libraries.
+
+[table:id Components
+    [[Boost.Utility] [Moved to Boost] [C++ Standard variant]]
+    [[[@../core/doc/html/core/addressof.html `addressof`]] [[@../core/index.html Boost.Core]] [C++11 __std_addressof__]]
```

Comment:
You can link to other boost libraries relative to the current commit or version using `@boost`. Here I link to an Asio page. This will always produce the correct URL as the Boost version or commit evolves:
```
[@boost:/doc/html/boost_asio/reference/strand.html `net::strand`]
```